src/main/java/persistence/sql/meta/ColumnFields.java

```diff
@@ -0,0 +1,58 @@
+package persistence.sql.meta;
+
+import org.h2.util.StringUtils;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ColumnFields {
+
+    private final List<ColumnField> columnFields;
+
+    public ColumnFields(Class<?> clazz) {
+        this.columnFields = extract(clazz);
+    }
+
+    private List<ColumnField> extract(Class<?> clazz) {
+        return Arrays.stream(clazz.getDeclaredFields())
+                .map(ColumnField::new)
+                .filter(ColumnField::isNotTransient)
+                .collect(Collectors.toList());
+    }
+
+    public List<ColumnField> getColumnFields() {
+        return columnFields;
+    }
```

Comment:
이왕 일급컬렉션을 만들어주셨는데 불변을 지킬 수 있으면 좋을 것 같아요.
개인적으로 일급컬렉션에 책임을 더 부여하는걸 선호하지만, getter 를 통해 외부로 노출해야할땐 불변 리스트로 반환 해볼 수 있어요 😃 
```suggestion
    public List<ColumnField> getColumnFields() {
        return Collections.unmodifiableList(columnFields);
    }
```
불변을 지키면 어떤 장점이 있을까요?